demo/src/entrypoint.ts

```diff
@@ -1,4 +1,5 @@
 import "../../src/resources/safari-14-attachshadow-patch";
+import "./util/is_frontpage";
```

Comment:
AI is correct here.  This import is doing nothing since the module has no side effects.

Reply:
It is working as expected though. The query string is stripped off by Lovelace and by evaluating early, we capture it.

Reply:
Oh right, forgot about that.  Sorry noticed while fixing conflict for #21247.